File: Assignment_2/Assignment2_2.py
"""-------------------------------------------------------------------------------------------
                          Assignment 2_2
                    (Student name - Vaishali Jorwekar)
                     Python by Marvellous Infosystems
---------------------------------------------------------------------------------------------
Problem statement : This program accepts a number from user and displays
                     number of * in following matrix format of number*number
                     e.g. Input :accepted number 5
                     Output * * * * *
                            * * * * *  
                            * * * * *
                            * * * * *
                            * * * * *  
-------------------------------------------------------------------------------------------"""
STAR ="*"
#---------------------------------------------------------------------------------------------
# This function accepts a parameter "number" and prints * in matrix format of number*number
# Used basic for loop
#------------------------------------------------------------------------------------------
def printStars(number):
    print(f"\nPrinting '*' in {number} * {number} matrix format")
    print("--------------------------------------------------------------------------\n")
    for row in range(number):
        for column in range(number):
              print(STAR,end="   ") 
        print()
    print("\n---------------------------------------------------------------------------")
#------------------------------------------------------------------------------------------
# This function accepts a input number from user and calls function 
# printStars(number) to print the stars in specific format
#------------------------------------------------------------------------------------------
def main():
    number=int(input("Enter number:"))
    
    printStars(number)
#-----------------------------------------------------------------------------------
# This is main entry point of the program
#-----------------------------------------------------------------------------------
if __name__=="__main__":
    main()
#-----------------------------------------------------------------------------------

File: JavaPractice/Assignment_2/Assignment2_2.java

import java.util.Scanner;
/*-------------------------------------------------------------------------------------------
			Assignment 2_2
			(Student name - Vaishali Jorwekar)
			Python by Marvellous Infosystems
---------------------------------------------------------------------------------------------
			Problem statement : This program accepts a number from user and displays
			number of * in following matrix format of number*number
						e.g. Input :accepted number 5
						Output * * * * *
 * * * * *  
 * * * * *
 * * * * *
 * * * * *  
-------------------------------------------------------------------------------------------*/


public class Assignment2_2 {
	static char STAR ='*';

	public static void printStars(int number) {
		System.out.println("Displaying * in "+number+"*"+number+" format");
		System.out.println("-------------------------------------------------");

		for(int row=1;row<=number;row++) {
			for(int column=1;column<=number;column++){
				System.out.print(STAR+" ");
			}
			System.out.println();
		}

		System.out.println("-------------------------------------------------");

	}
	public static void printStarsLoop(int number) {
		System.out.println("Displaying * in "+number+"*"+number+" format");
		System.out.println("-------------------------------------------------");
		int loopCounter=AssignmentModule.Multiply(number, number);

		for(int row=1;row<=loopCounter;row++) {
			if(row%number!=0) {
				System.out.print(STAR+" ");
			}
			else {
				System.out.print(STAR+"\n");
			}
		}
		System.out.println();


		System.out.println("-------------------------------------------------");

	}
	public static void main(String args[]) {
		Scanner inputObj=new Scanner(System.in);
		System.out.println("Enter the number to display * in matrix format:");
		int number=inputObj.nextInt();
		printStars(number);
		printStarsLoop(number);
		inputObj.close();
	}
}
